packs/javascript-yarn/charts/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+name: REPLACE_ME_APP_NAME
+description: A Helm chart for Kubernetes
+apiVersion: v1
+version: 0.1.0-SNAPSHOT
+icon: https://raw.githubusercontent.com/cdfoundation/artwork/master/jenkinsx/icon/color/jenkinsx-icon-color.png
```

Comment:
A minor thing, but I think the yarn icon would be more fitting.

Reply:
updated